README.md

```diff
@@ -1,17 +1,12 @@
-# Fauna Go
-
-> **Note**
-> This driver is in beta release and not recommended for production use. It operates with the Fauna database service via an API which is also in beta release, and is not recommended for production use. This driver is not compatible with v4 or earlier versions of Fauna. If you would like to participate in the private beta program please contact product@fauna.com.
-
-Go driver for Fauna
-
 # Fauna Go Driver
-
 [![Go Report Card](https://goreportcard.com/badge/github.com/fauna/fauna-go)](https://goreportcard.com/report/github.com/fauna/fauna-go)
 [![Go Reference](https://pkg.go.dev/badge/github.com/fauna/fauna-go.svg)](https://pkg.go.dev/github.com/fauna/fauna-go)
 [![License](https://img.shields.io/badge/license-MPL_2.0-blue.svg?maxAge=2592000)](https://raw.githubusercontent.com/fauna/fauna-go/main/LICENSE)
 
-A Go lang driver for [Fauna](https://fauna.com/).
+A Golang driver for [Fauna](https://fauna.com/).
+
+> **Note**
+> This driver is in beta release and not recommended for production use. It operates with the Fauna database service via an API which is also in beta release, and is not recommended for production use. This driver is not compatible with v4 or earlier versions of Fauna. If you would like to participate in the private beta program please contact product@fauna.com.
```

Comment:
I recommend keeping this at the very top, and updating to the following. It's harder to avoid because it breaks normal formatting, like here https://github.com/fauna/fauna-js/blob/main/README.md

```suggestion
> **WARNING**
> This driver is in beta release and not recommended for production use. It operates with the Fauna database service via an API which is also in beta release, and is not recommended for production use. This driver is not compatible with v4 or earlier versions of Fauna. If you would like to participate in the private beta program please contact product@fauna.com.
```